Rails/Ruby (Rails).sublime-settings

```diff
@@ -0,0 +1,5 @@
+{
+  // This gives precedence over the first file_extensions entry in the
+  // "Ruby (Rails).sublime-syntax" file.
+  "default_extension": ["rb"]
```

Comment:
This seems should be `"default_extension": "rb"` ?

Reply:
Oh my... too many things around, sorry. Force pushed the correct version.